settings.gradle.kts

```diff
@@ -1,11 +1,11 @@
 pluginManagement {
-    repositories {
-	    gradlePluginPortal()
-    }
+	repositories {
+		gradlePluginPortal()
+	}
 }
 
 plugins {
-	id("org.gradle.toolchains.foojay-resolver-convention") version("0.7.0")
+	id("org.gradle.toolchains.foojay-resolver-convention") version ("0.7.0")
```

Comment:
Is `version` not a function here?? Kotlin syntax never ceases to find new ways to disgust me...